git.md

```diff
@@ -186,19 +206,19 @@ e323208 Hub - deprecate `primary` for `primary_key`
 326cc64 Hub - bug fix of ISO date detected as IP, also:
 ```
 
-Based on this, combined with a good convention [on commits](#on-commits), we can leverage tools like (our own) [release CLI](https://github.com/EQWorks/release) to generate nicely arranged release notes material.
+Based on this, combined with a good convention [on commits](#on-commits), we can leverage tools like (our own) [`release` CLI](https://github.com/EQWorks/release) to generate nicely arranged release notes material.
 
 ## On Pull/Merge Requests
 
 Try to maintain a good length and depth of the code changes within each request, make it digestible within 30 minutes or less.
 
-Try to keep a [`CHANGELOG`](https://keepachangelog.com)-like summaries of changes in the request description. Try to attach screenshots/videos to demonstrate the workflow and before/after differences. This can be automated/semi-automated through good practices [on commits](#on-commits).
+Try to keep a concise summaries of changes in the request description. Try to attach screenshots/videos to demonstrate the workflow and before/after differences. A good practices [on commits](#on-commits) helps tremendously on conveying the summary of changes, so rebase/squash/rephrase commits as needed to keep the commit history clean.
 
-Reference and mark relevant issues as fixed (`fixes #123`), or label the request itself when there are no relevant issues -- not both.
+Reference and mark relevant issues as fixed/closed (e.g.: `fixes #123`) when applicable.
 
-Start the request as a draft and/or mark in the title with `[WIP]` prefix when it needs more time to complete. Mark it ready to review, and/or with a `[G2M]` prefix in the title to signal its readiness.
+Start the request as a draft and/or mark in the title with a `[WIP]` prefix when it needs more time to complete. Mark it ready to review, and/or with a `[G2M]` prefix in the title to signal its readiness.
 
-When reviewer approved the PR, it's ready for merge. Three ways to meget it through GitHub:
+When reviewer approved the PR, it's ready for merge. There are three ways to meget it through GitHub:
```

Comment:
suggestions:
- should be 'When a reviewer or reviewers approves the PR, it's ready for merging.'
- should be 'There are three ways to merge it through Github'